packages/code-du-travail-frontend/src/layout/Headband.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Link from "next/link";
+import { AlertCircle } from "react-feather";
+import { IconStripe, theme } from "@socialgouv/react-ui";
+import styled from "styled-components";
+
+import { HEADER_HEIGHT, MOBILE_HEADER_HEIGHT } from "./Header";
+
+export const Headband = ({ currentPage }) => {
+  if (
+    currentPage &&
```

Comment:
why not use useRouter asPath property here ?

Reply:
100% agreed